pkg/cloud/services/networking/service.go

```diff
@@ -92,6 +92,10 @@ func (s *Service) replaceAllAttributesTags(eventObject runtime.Object, resourceT
 		panic(fmt.Errorf("invalid argument: resourceType, %s, does not match allowed arguments: %s or %s", resourceType, trunkResource, portResource))
 	}
 
+	if len(tags) == 0 {
+		record.Warnf(eventObject, "IgnoreReplaceAllAttributesTags", "no tags provided: , %s", resourceID)
```

Comment:
looks good and I guess  a `info` is reasonable ... as it's normal no tags input 

or we can check whether tags is empty before call this function , both works

Reply:
Moved it to the stop. 
on the `info`, I was thinking the same. but, no such function for `record`
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/pkg/record/recorder.go